cmd/ical-relay/main.go

```diff
@@ -10,7 +12,8 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-var version = "2.0.0-beta.7.3"
+//go:embed VERSION
+var version string
```

Comment:
```suggestion
var version string
// If you are here due to a compile error, run go generate
```

Reply:
I think this might help avoid some head scratching in the future. However, with the go generate comment above it is somewhat obvious.